docker/airflow/1.10.5/Dockerfile

```diff
@@ -27,6 +27,7 @@ LABEL io.astronomer.docker.airflow.version="1.10.5"
 ARG ORG="astronomer"
 ARG VERSION="1.10.5-2"
 ARG SUBMODULES="all, statsd, elasticsearch"
+ARG REPO_BRANCH=master
```

Comment:
@ashb it feels a little weird to be always pulling artifacts from our `master` branch. Like if we're building from a release branch, we'd want to install from that branch probably right?

Probably not the biggest deal, just wanted to discuss.

Reply:
Yes, this was the change I wanted to make after #514 was made.

Reply:
Now done via `bin/build-airflow`